tests/CoreFX/CoreFX.issues.json

```diff
@@ -751,6 +751,10 @@
                     "name": "System.Buffers.Text.Tests.ParserTests.TestParserDouble",
                     "reason": "https://github.com/dotnet/coreclr/pull/20707"
                 },
+                {
```

Comment:
I do not think you need to do this. The Index/Range tests are not in the corefx snapshot used by the CoreCLR CI.

Reply:
Interesting, I saw some failures in Range tests in previous runs. maybe the tests are coming from other places (e.g. jit for example)? I'll look more at that to understand what is going on.

Reply:
yes, it looks JIT tests. I'll try to find it and fix them then.

Reply:
it is the test https://github.com/dotnet/coreclr/blob/398c3bc97d5297df9773284a03e425f90c40bcf0/tests/src/JIT/Regression/JitBlue/GitHub_20958/GitHub_20958.cs#L14 I'll fix it and revert the stuff added to corefx.issues.json. thanks @jkotas